### ISSUE TITLE
[FEATURE REQUEST] set config-file and defaults-config-file differently

### ISSUE THREAD

**Issue by nyxnor** [nyxnor/onionjuggler#79]
**Is your feature request related to a problem? Please describe.**

currently, it tries to read the tor@default.service file of systemd.
But not every system has this file and not everybody uses systemd.

```sh
tor-ctrl -m GETINFO config-file config-defaults-file
tor-prompt --run "GETINFO config-file config-defaults-file"
carml cmd GETINFO config-file config-defaults-file
```
Interacting with the tor controller requires one of these tools.
But pratically noone installs them:
- tor-ctrl -> available by default on whonix and the fasted option
- tor-prompt -> stem is not being developed anymore
- carml -> really great but not as famous as tor-prompt and takes 3 times more to run

**Describe the solution you'd like**

Setting on the onionjuggler configuration would be the cleanest way, this way, it won't need to think how to get the values, it already has it.

**Describe alternatives you've considered**

**Additional context**

**Comment by nyxnor** [comment]
To set the variable name, I need some ideas, because `tor_conf` is already taken.

https://gitweb.torproject.org/torspec.git/tree/control-spec.txt#n673

    "config-file" -- The location of Tor's configuration file ("torrc").

    "config-defaults-file" -- The location of Tor's configuration
      defaults file ("torrc.defaults").  This file gets parsed before
      torrc, and is typically used to replace Tor's default
      configuration values. [First implemented in 0.2.3.9-alpha.]

**man torrc**

       -f, --torrc-file FILE
           Specify a new configuration file to contain further Tor configuration options, or pass - to make Tor read its
           configuration from standard input. (Default: /etc/tor/torrc, or $HOME/.torrc if that file is not found.)

       --allow-missing-torrc
           Allow the configuration file specified by -f to be missing, if the defaults-torrc file (see below) is accessible.

       --defaults-torrc FILE
           Specify a file in which to find default values for Tor options. The contents of this file are overridden by those in
           the regular configuration file, and by those on the command line. (Default: /etc/tor/torrc-defaults.)

       --ignore-missing-torrc
           Specify that Tor should treat a missing torrc file as though it were empty. Ordinarily, Tor does this for missing
           default torrc files, but not for those specified on the command line.

---------

tor_defaults_conf
tor_torrc_conf
tor_conf

**Comment by nyxnor** [comment]
Another thing comes to mind.
If user has no defaults torrc, and only uses one torrc, for example `/etc/tor/torrc`, then safe_edit will change the original file to `.torrc` and the temp `torrc.XXXXXX.tmp`

So the commands:

 `tor --verify-config -f /etc/tor/torrc --ignore-missing-torrc`
```
[notice] Configuration file "/etc/tor/torrc" not present, using reasonable defaults.
Configuration was valid
```
 `tor --verify-config -f /etc/tor/torrc`
```
[warn] Unable to open configuration file "/etc/tor/torrc".
[err] Reading config failed--see warnings above.
```
